charts/fission-all/templates/deployment.yaml

```diff
@@ -128,7 +128,11 @@ spec:
     spec:
       containers:
       - name: controller
+{{ if .Values.repository }}
```

Comment:
I think it would be more convenient to define a template in `_helpers.tpl` and include it multiple times.

Reply:
sounds good. I'll do that by tomorrow.